DataModel/ExperimentDesign/ExperimentDesign-definition.yml

```diff
@@ -7,6 +7,13 @@ ExperimentDesign:
       description: The organism under study, providing information about the species or biological system.
       format: NCBITaxon
       type: string
+    cellLine:
```

Comment:
@ishitasati11 Let's also add a field called `pattern` with each attribute where certain rules can be applied for better validation. For instance all cell line IDs must start with `CVCL` so it is assigned a regex pattern.